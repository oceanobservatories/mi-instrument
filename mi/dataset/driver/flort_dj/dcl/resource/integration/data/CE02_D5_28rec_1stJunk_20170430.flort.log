2017/04/30 00:00:12.584 [flort:DLOGP2]:Instrument sampling - Data logger entering data acquisition state
2017/04/30 00:00:12.621 29/17	23:58:59	700	554	695	170	460	66	556
2017/04/30 00:00:13.745 04/29/17	23:59:00	700	406	695	181	460	66	556
2017/04/30 00:00:14.874 04/29/17	23:59:01	700	411	695	183	460	64	556
2017/04/30 00:01:56.511 04/30/17	00:00:43	700	698	695	184	460	65	553
2017/04/30 00:01:57.640 04/30/17	00:00:44	700	493	695	201	460	62	553
2017/04/30 00:01:58.769 04/30/17	00:00:45	700	510	695	194	460	64	553
2017/04/30 00:02:17.958 04/30/17	00:01:04	700	451	695	208	460	64	552
2017/04/30 00:02:19.126 04/30/17	00:01:06	700	436	695	178	460	65	552
2017/04/30 00:02:20.216 04/30/17	00:01:07	700	366	695	184	460	64	552
2017/04/30 00:02:21.345 04/30/17	00:01:08	700	433	695	191	460	63	552
2017/04/30 00:02:34.893 04/30/17	00:01:21	700	341	695	200	460	66	552
2017/04/30 00:02:47.312 04/30/17	00:01:34	700	493	695	177	460	66	552
2017/04/30 00:03:01.989 04/30/17	00:01:48	700	418	695	189	460	69	552
2017/04/30 00:03:02.196 [flort:DLOGP2]:Instrument Stopped [Power Off]
2017/04/30 00:30:13.886 [flort:DLOGP2]:Instrument sampling - Data logger entering data acquisition state
2017/04/30 00:30:13.924 30/17	00:29:00	700	406	695	170	460	66	557
2017/04/30 00:30:15.051 04/30/17	00:29:02	700	495	695	165	460	66	557
2017/04/30 00:30:16.181 04/30/17	00:29:03	700	391	695	165	460	64	556
2017/04/30 00:31:43.131 04/30/17	00:30:30	700	384	695	168	460	65	553
2017/04/30 00:31:55.553 04/30/17	00:30:42	700	435	695	172	460	63	553
2017/04/30 00:32:23.784 04/30/17	00:31:10	700	342	695	178	460	64	552
2017/04/30 00:33:02.387 [flort:DLOGP2]:Instrument Stopped [Power Off]
2017/04/30 01:00:13.177 [flort:DLOGP2]:Instrument sampling - Data logger entering data acquisition state
2017/04/30 01:00:13.213 30/17	00:59:00	700	381	695	172	460	62	557
2017/04/30 01:01:24.351 04/30/17	01:00:11	700	334	695	156	460	63	554
2017/04/30 01:01:48.061 04/30/17	01:00:35	700	350	695	180	460	64	554
2017/04/30 01:02:11.770 04/30/17	01:00:58	700	444	695	178	460	64	553
2017/04/30 04:33:01.763 [flort:DLOGP2]:Instrument Stopped [Power Off]
