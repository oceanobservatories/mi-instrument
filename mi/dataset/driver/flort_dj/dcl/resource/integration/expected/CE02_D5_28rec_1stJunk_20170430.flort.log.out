
 subsite  | node  | sensor       | bin        | method      | time                  | deployment | internal_timestamp | port_timestamp        | preferred_timestamp | quality_flag | date_string | dcl_controller_timestamp | measurement_wavelength_beta | measurement_wavelength_cdom | measurement_wavelength_chl | raw_internal_temp | raw_signal_beta | raw_signal_cdom | raw_signal_chl | time_string
----------+-------+--------------+------------+-------------+-----------------------+------------+--------------------+-----------------------+---------------------+--------------+-------------+--------------------------+-----------------------------+-----------------------------+----------------------------+-------------------+-----------------+-----------------+----------------+-------------
 CE02SHSM | RID27 | 02-FLORTD000 | 3702499200 | telemetered | 3702499213.7449998856 |          5 |         3702499140 | 3702499213.7449998856 |      port_timestamp |           ok |        null |                     null |                         700 |                         460 |                        695 |               556 |             406 |              66 |            181 |        null
 CE02SHSM | RID27 | 02-FLORTD000 | 3702499200 | telemetered | 3702499214.8740000725 |          5 |         3702499141 | 3702499214.8740000725 |      port_timestamp |           ok |        null |                     null |                         700 |                         460 |                        695 |               556 |             411 |              64 |            183 |        null
 CE02SHSM | RID27 | 02-FLORTD000 | 3702499200 | telemetered | 3702499316.5109996796 |          5 |         3702499243 | 3702499316.5109996796 |      port_timestamp |           ok |        null |                     null |                         700 |                         460 |                        695 |               553 |             698 |              65 |            184 |        null
 CE02SHSM | RID27 | 02-FLORTD000 | 3702499200 | telemetered | 3702499317.6400003433 |          5 |         3702499244 | 3702499317.6400003433 |      port_timestamp |           ok |        null |                     null |                         700 |                         460 |                        695 |               553 |             493 |              62 |            201 |        null
 CE02SHSM | RID27 | 02-FLORTD000 | 3702499200 | telemetered | 3702499318.7690000534 |          5 |         3702499245 | 3702499318.7690000534 |      port_timestamp |           ok |        null |                     null |                         700 |                         460 |                        695 |               553 |             510 |              64 |            194 |        null
 CE02SHSM | RID27 | 02-FLORTD000 | 3702499200 | telemetered | 3702499337.9580001831 |          5 |         3702499264 | 3702499337.9580001831 |      port_timestamp |           ok |        null |                     null |                         700 |                         460 |                        695 |               552 |             451 |              64 |            208 |        null
 CE02SHSM | RID27 | 02-FLORTD000 | 3702499200 | telemetered | 3702499339.1259999275 |          5 |         3702499266 | 3702499339.1259999275 |      port_timestamp |           ok |        null |                     null |                         700 |                         460 |                        695 |               552 |             436 |              65 |            178 |        null
 CE02SHSM | RID27 | 02-FLORTD000 | 3702499200 | telemetered | 3702499340.2160000801 |          5 |         3702499267 | 3702499340.2160000801 |      port_timestamp |           ok |        null |                     null |                         700 |                         460 |                        695 |               552 |             366 |              64 |            184 |        null
 CE02SHSM | RID27 | 02-FLORTD000 | 3702499200 | telemetered |  3702499341.345000267 |          5 |         3702499268 |  3702499341.345000267 |      port_timestamp |           ok |        null |                     null |                         700 |                         460 |                        695 |               552 |             433 |              63 |            191 |        null
 CE02SHSM | RID27 | 02-FLORTD000 | 3702499200 | telemetered |  3702499354.892999649 |          5 |         3702499281 |  3702499354.892999649 |      port_timestamp |           ok |        null |                     null |                         700 |                         460 |                        695 |               552 |             341 |              66 |            200 |        null
 CE02SHSM | RID27 | 02-FLORTD000 | 3702499200 | telemetered | 3702499367.3120002747 |          5 |         3702499294 | 3702499367.3120002747 |      port_timestamp |           ok |        null |                     null |                         700 |                         460 |                        695 |               552 |             493 |              66 |            177 |        null
 CE02SHSM | RID27 | 02-FLORTD000 | 3702499200 | telemetered | 3702499381.9890003204 |          5 |         3702499308 | 3702499381.9890003204 |      port_timestamp |           ok |        null |                     null |                         700 |                         460 |                        695 |               552 |             418 |              69 |            189 |        null
 CE02SHSM | RID27 | 02-FLORTD000 | 3702499200 | telemetered | 3702501015.0510001183 |          5 |         3702500942 | 3702501015.0510001183 |      port_timestamp |           ok |        null |                     null |                         700 |                         460 |                        695 |               557 |             495 |              66 |            165 |        null
 CE02SHSM | RID27 | 02-FLORTD000 | 3702499200 | telemetered | 3702501016.1809997559 |          5 |         3702500943 | 3702501016.1809997559 |      port_timestamp |           ok |        null |                     null |                         700 |                         460 |                        695 |               556 |             391 |              64 |            165 |        null
 CE02SHSM | RID27 | 02-FLORTD000 | 3702499200 | telemetered |  3702501103.131000042 |          5 |         3702501030 |  3702501103.131000042 |      port_timestamp |           ok |        null |                     null |                         700 |                         460 |                        695 |               553 |             384 |              65 |            168 |        null
 CE02SHSM | RID27 | 02-FLORTD000 | 3702499200 | telemetered | 3702501115.5529999733 |          5 |         3702501042 | 3702501115.5529999733 |      port_timestamp |           ok |        null |                     null |                         700 |                         460 |                        695 |               553 |             435 |              63 |            172 |        null
 CE02SHSM | RID27 | 02-FLORTD000 | 3702499200 | telemetered | 3702501143.7839999199 |          5 |         3702501070 | 3702501143.7839999199 |      port_timestamp |           ok |        null |                     null |                         700 |                         460 |                        695 |               552 |             342 |              64 |            178 |        null
 CE02SHSM | RID27 | 02-FLORTD000 | 3702499200 | telemetered | 3702502884.3509998322 |          5 |         3702502811 | 3702502884.3509998322 |      port_timestamp |           ok |        null |                     null |                         700 |                         460 |                        695 |               554 |             334 |              63 |            156 |        null
 CE02SHSM | RID27 | 02-FLORTD000 | 3702499200 | telemetered | 3702502908.0609998703 |          5 |         3702502835 | 3702502908.0609998703 |      port_timestamp |           ok |        null |                     null |                         700 |                         460 |                        695 |               554 |             350 |              64 |            180 |        null
 CE02SHSM | RID27 | 02-FLORTD000 | 3702499200 | telemetered | 3702502931.7699999809 |          5 |         3702502858 | 3702502931.7699999809 |      port_timestamp |           ok |        null |                     null |                         700 |                         460 |                        695 |               553 |             444 |              64 |            178 |        null

(20 rows)
